tests/phpunit/helpers/class-wc-subscription.php

```
@@ -143,6 +143,15 @@ public function set_status( $status, $note = '', $manual_update = false ) {
 		$this->status = $status;
 	}
 
+	/**
+	 * @inheritDoc
+	 * @return bool
+	 */
+	public function update_status( $new_status, $note = '', $manual = false ) {
```

PHPMD (2.15.0)

[error] The method update_status has a boolean flag argument $manual, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)